jupyterhub/apihandlers/users.py

```diff
@@ -209,7 +213,7 @@ def post(self, name):
             raise web.HTTPError(400, "%s's server is not running" % name)
         self.set_server_cookie(user)
         # a service can also ask for a user cookie
-        # this code prevent to raise an error
+        # this code prevents to raise an error
```

Comment:
@minrk: For my own understanding. Is this just preventing an attribute error if `current.other_user_cookies` does not exist?

`# this code prevents raising an AttributeError`